### PR TITLE
[Console-IoT] Rename use of 'dropdowntoggle' in ID to 'dropdown'

### PR DESCRIPTION
**Pull request by suyash-naithani**
### Type of change

- Refactoring

### Description

 Remove error messages data from project list
     Rename use of dropdowntoggle in ID to dropdown

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by suyash-naithani** [comment]
Please have a look at the latest commit. Will be fixing the extra commits showing up in this PR

File: console/console-init/ui/src/modules/project/components/ProjectList/ProjectList.tsx [hunk 1]
```diff
@@ -20,7 +20,6 @@ import { StyleSheet, css } from "aphrodite";
 import {
   StatusTypes,
   ProjectStatus,
-  ProjectError,
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/modules/project/components/ProjectList/ProjectList.tsx:23]: @suyash-naithani I already raised a PR https://github.com/EnMasseProject/enmasse/pull/5075 to comment out the error count from  the project list. Could you please undo these changes?

Comment (suyash-naithani) [reply console/console-init/ui/src/modules/project/components/ProjectList/ProjectList.tsx:23]: @drcoolsanjeev Okay, will do

Comment (suyash-naithani) [reply console/console-init/ui/src/modules/project/components/ProjectList/ProjectList.tsx]: @drcoolsanjeev Removed the error count conflict. This PR is solely for updating  minor changes to ID's for the project
